package.json

```
@@ -1,38 +1,118 @@
 {
-  "name": "@storyblok/react-workspace",
-  "version": "0.0.1",
+  "name": "@storyblok/react",
+  "type": "module",
+  "version": "4.0.0",
   "private": false,
+  "packageManager": "pnpm@9.13.2",
+  "description": "SDK to integrate Storyblok into your project using React.",
+  "author": "Storyblok",
   "license": "MIT",
-  "workspaces": [
-    "./lib",
-    "./playground",
-    "./playground-next",
-    "./playground-next13-rsc",
-    "./playground-next13-live-editing"
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/storyblok/storyblok-react"
+  },
+  "bugs": {
+    "url": "https://github.com/storyblok/storyblok-react/issues"
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js"
+    },
+    "./rsc": {
+      "types": "./dist/rsc/index.d.ts",
+      "import": "./dist/rsc.mjs",
+      "require": "./dist/rsc.js"
+    }
+  },
+  "main": "./dist/index.js",
+  "module": "./dist/index.mjs",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist"
   ],
   "scripts": {
-    "dev": "npm run dev --workspace=lib & npm run demo",
-    "demo": "npm run demo --workspace=playground",
-    "demo-next": "npm run build && npm run demo --workspace=playground-next",
-    "demo-next13-rsc": "npm run demo --workspace=playground-next13-rsc",
-    "demo-next13-live-editing": "npm run demo --workspace=playground-next13-live-editing",
-    "build": "npm run build --workspace=lib",
-    "test": "npm run test --workspace=lib",
-    "test:unit": "npm run test:unit --workspace=lib",
-    "test:e2e": "npm run test:e2e --workspace=lib",
+    "dev": "vite build --watch",
+    "build": "vite build && tsc --project tsconfig.json",
+    "test": "pnpm run test:unit && pnpm run test:e2e",
+    "test:unit": "vitest",
+    "test:unit:ci": "vitest run",
+    "test:unit:ui": "vitest --ui",
+    "test:e2e": "start-server-and-test playground-next13-app-router http://localhost:3000/ cy:run",
+    "test:e2e-watch": "start-server-and-test playground-next13-app-router http-get://localhost:3000/ cy:open",
+    "cy:run": "cypress run",
+    "cy:open": "cypress open",
+    "test:comp": "cypress run-ct",
+    "playground": "pnpm run --filter ./playground/react dev",
+    "playground-next": "pnpm run --filter ./playground/next15 dev",
```

Non blocking:

To keep consistent script conventions.

```suggestion
    "playground:next": "pnpm run --filter ./playground/next15 dev",
```